tensorflow/core/kernels/dml_reduce_ops.cc

```diff
@@ -175,11 +175,10 @@ class DmlReduceKernel : public DmlKernel {
       DmlKernelTensors tensors;
       tensors.outputs.resize(1);
 
-      constexpr uint32_t dml_output_shape[] = {1u};
-
       tensors.outputs[0].emplace();
-      tensors.outputs[0]->desc = DmlTensorDesc::Create(
-          output_type, dml_output_shape, dml_output_shape);
+      tensors.outputs[0]->desc =
+          DmlTensorDesc::Create(output_type, ctx->GetOutputTensorShape(0),
```

Comment:
I assume the output shape might be something like `[1,1,1,1,1,1,1,1,1,1,1]`? It seems like we may want some shape simplification for the purpose of DML's tensor desc to ensure we don't run into a situation where the op can't be created.